src/app/launcher/import-app/gitprovider-importapp-step/gitprovider-importapp-step.component.html

```diff
@@ -4,9 +4,6 @@
     <div class="container-fluid">
       <h1 snInViewport (onInViewportChange)="launcherComponent.onInViewportChange($event, 'GitProvider')"
           [debounce]="500">Authorize Git Provider</h1>
-      <p>
```

Comment:
it's there as per UXD https://redhat.invisionapp.com/share/VGF1RF15J#/screens/286893664

Reply:
Per the copy docs (https://docs.google.com/document/d/1PRKPlg9zOgG9Eu5GZr96GCADPhPnLwZte0jkVx5q3KQ/edit?usp=sharing), that description should not be present. The copy docs are the final representation of what the text of the mockups should be. 120 screens is too much to keep updated text on, hence the copy docs :)

@catrobson @bdellasc